### DISCUSSION TITLE
Fluxnews: Offline viewing option for Video posts

### DISCUSSION THREAD

**Discussion by shailendersindhu** [inspireui/support#5642] [New Feature/Ideas Requests]
Can we add a feature to give option to users to download video for offline viewing ? 

This will be helpful in following scenarios:
1. When a long video post is uploaded.
2. Users can download/save it and then watch it without internet for e.g. in flight.
3. Downloaded over wifi and watch/listen while travelling

It can be a key differentiator for the app. 

Thanks in advance.

**Comment by anjuSindhu** [comment]
I also feel it's an important feature.
